code/game/gamemodes/freeze/structures.dm

```diff
@@ -0,0 +1,43 @@
+//turf frost - these are children of alien weeds because they're basically identical
+//exposure to air above T20C will slowly melt them, wetting the floor as they melt
+/obj/structure/alien/weeds/frost
```

Comment:
you know it's kind of freaky how similar my implementation would have been to yours.